barretenberg/cpp/src/barretenberg/common/fuzzer.hpp

```diff
@@ -2,7 +2,7 @@
 #include "barretenberg/circuit_checker/circuit_checker.hpp"
 #include "barretenberg/numeric/uint256/uint256.hpp"
 #include "barretenberg/stdlib_circuit_builders/standard_circuit_builder.hpp"
-#include <concepts>
+#include "barretenberg/stdlib_circuit_builders/ultra_circuit_builder.hpp"
```

Comment:
Why do you need to include this header here?

Reply:
`RunWithBuilders` function uses ultra circuit. But I guess we can leave only the `circuit_checker` include